test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/DefaultEditorTemplatesTest.cs

```diff
@@ -1076,6 +1082,8 @@ public void Editor_InputTypeDateTime_RendersAsDateTime()
         [InlineData("datetime-local", Html5DateRenderingMode.Rfc3339, "datetime-local")]
         [InlineData("time", Html5DateRenderingMode.CurrentCulture, "time")]
         [InlineData("time", Html5DateRenderingMode.Rfc3339, "time")]
+        [InlineData("month", Html5DateRenderingMode.CurrentCulture, "month")]
```

Comment:
I'm not sure we need to render the dates on the current culture. These inputs are HTML5 specific and the current culture rendering mode is probably not valid for them. So I think they should render in Rfc3339 mode always. /cc: @dougbu

Reply:
RFC 3339 mode is the default. But, we should consistently switch to locale-specific formats when that mode is disabled.

Put another way, be consistent with `type="time"` et cetera.

Reply:
> be consistent with `type="time"`

That is what I was going for too.